index.html

```diff
@@ -0,0 +1,200 @@
+<!DOCTYPE html>
+<html lang="ko">
```

Comment:
"적용해도 좋고 안해도 좋아요"

`ko`로 하여도 한국어를 표현하는데에 전혀 문제가 없지만, 더 정확하게는 남한을 `ko-KR` 과 북한을 뜻하는`ko-KP`로 나눌 수 있어요!
```suggestion
<html lang="ko-KR">
```

Reply:
좋은 의견입니다!! 동의합니다. 적극 반영하겠습니다